bower.json

```diff
@@ -0,0 +1,17 @@
+{
+  "name": "customapps-medallia-library-FileSaver.js",
+  "version": "0.0.1",
```

Comment:
I know it's not semver, but just a thought on versioning numbers here.  We could always use the upstream version as a base and append a medallia-specific version to it, much like is used in Debian packaging.  Example:

```
1.1.20151003-medallia-1
```

Effectively, we would just monotonically increment the `-1` suffix, unless we sync with upstream changes.